### PR TITLE
#571: Add missing mdu keywords

### PR DESCRIPTION
**Pull request by tim-vd-aardweg**
Refs: #571

### CODE REVIEW

File: hydrolib/core/dflowfm/mdu/models.py [hunk 1]
```diff
@@ -1263,6 +1400,75 @@ class Comments(INIBasedModel.Comments):
...
+            "Write boundary points to map file (1: yes, 0: no).", alias="wrimap_bnd"
+        )
+        wrishp_dambreak: Optional[str] = Field(
+            "Writing dambreaks to shape file (0=no, 1=yes).", alias="wrishp_dambreak"
+        )
+        wrimap_waterdepth_hu: Optional[str] = Field(
+            "Write water depths on u-points to map file (1: yes, 0: no).",
+            alias="wrimap_waterdepth_hu",
+        )
+        ncmapdataprecision: Optional[str] = Field(
```
Comment (MRVermeulenDeltares) [inline hydrolib/core/dflowfm/mdu/models.py:1450]: this is an int below, but this is talking about precision, should it be an int?

File: hydrolib/core/dflowfm/mdu/models.py [hunk 2]
```diff
@@ -568,6 +704,7 @@ class Comments(INIBasedModel.Comments):
     pavbnd: float = Field(0.0, alias="pavBnd")
     pavini: float = Field(0.0, alias="pavIni")
     computedairdensity: bool = Field(False, alias="computedAirdensity")
+    stresstowind: float = Field(0.0, alias="stressToWind")
```
Comment (MRVermeulenDeltares) [inline hydrolib/core/dflowfm/mdu/models.py:707]: should the alias be with capital letters?

File: hydrolib/core/dflowfm/mdu/models.py [hunk 3]
```diff
@@ -553,6 +685,10 @@ class Comments(INIBasedModel.Comments):
         computedairdensity: Optional[str] = Field(
             "Compute air density yes/no (), 1/0, default 0.", alias="computedAirdensity"
         )
+        stresstowind: Optional[str] = Field(
+            "Switch between Wind speed (=0) and wind stress (=1) approach for wind forcing.",
```
Comment (MRVermeulenDeltares) [inline hydrolib/core/dflowfm/mdu/models.py:689]: is this text correct? it sounds like a bool in this case, or am I interperting this incorrectly?

File: hydrolib/core/dflowfm/mdu/models.py (hunk 1 above)
Comment (tim-vd-aardweg) [reply hydrolib/core/dflowfm/mdu/models.py:1450]: I assume it is an `int`. The default value according to the very up-to-date manual is: `0` (an integer).

EDIT: I just checked the file Jelmer provided in the issue (the allow_extra file), there the description of this keyword more clearly indicates that it is indeed an integer with two valid values 0 and 1.

`Precision for NetCDF data in map files (0: double, 1: float)`

File: hydrolib/core/dflowfm/mdu/models.py (hunk 3 above)
Comment (tim-vd-aardweg) [reply hydrolib/core/dflowfm/mdu/models.py:689]: According to the manual's default value, it's a float/double. Reading the description, it made more sense to make it an int with only two options: 0 or 1.

**Review by MRVermeulenDeltares — APPROVED** [review]

**Pull request merged**